.rubocop.yml

```diff
@@ -80,3 +80,7 @@ RSpec/ExampleLength:
 
 RSpec/MultipleExpectations:
   Enabled: false
+
+# as discussed in issue#144
+Style/ParenthesesAroundCondition:
```

Comment:
@mbarnett  already fixed this in Master haha. Not sure if that was clear sorry alex.

https://github.com/ualbertalib/Hydranorth2/blob/master/.rubocop.yml#L74-L75